README.md

```diff
@@ -1,4 +1,7 @@
-# Confita
+[](https://user-images.githubusercontent.com/4570448/40786144-a8e8ce34-64ea-11e8-90b9-cb838cd13b88.png)
```

Comment:
You should use the following notation:

`![logo](img url)`

Reply:
I didn't because I want to resize the image. There is a lot of solution on the internet which didn't worked for me except this one.